index.html

```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
+    </h2>
+    <form class="form" action="https://formspree.io/html/" method="post">
+      <div class="input-username">
+        <input
+      class="input-1"
+      type="text" value="Arnold"
+      maxlength="30"
+      placeholder="Full name"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      <div class="input-username">
+        <input 
+      class="input-1"
+      type="email"
+      placeholder="Email address"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      
+      <textarea class="textarea" maxlength="500">Enter text here</textarea>
+      <button class="btn-submit" type="submit">Get in touch</button>
+    </form>
+  </section>
+  <footer class="footer-last">
+      <ul class="brands1">
+       <li class="icon">
+        <a href="#"><img src="./Images/github.png" alt="github"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/linkedIn.png" alt="linkedIn"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/peace.png" alt="Angellist"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/twitter.png" alt="twitter"></a>
+       </li>
+       <li class="icon">
+        <a href="#"><img src="./Images/m.png" alt="m"></a>
+       </li>
+      </ul>  
```

Comment:
- [ ] You did well 😃here at adding the icons, but please try to stick to the design as I see here that the size of the icons is bigger than the Figma design, so please try to fix that to be like the Figma Design 😊
- [ ] Also please try to add the hr at the top of the icons to be like the Figma design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|![image](https://user-images.githubusercontent.com/83360636/221977161-1df23ab7-2324-4909-9e8b-4333f0c017e6.png)   |    ![image](https://user-images.githubusercontent.com/83360636/221977080-b42f7d77-683d-4e1d-a64e-b9335a860434.png) |